engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSender.java

```diff
@@ -77,6 +78,38 @@ public void init() {
         bopApiToken = LineBreakCleaner.clean(bopApiToken);
     }
 
+    public void sendEmail(Set<User> user, Event event, TemplateInstance subject, TemplateInstance body, boolean persistHistory) {
+        final HttpRequest<Buffer> bopRequest = this.buildBOPHttpRequest();
+        LocalDateTime start = LocalDateTime.now(UTC);
+
+        Action action = event.getAction();
+
+        Timer.Sample processedTimer = Timer.start(registry);
+
+        // uses canonical EmailSubscription
+        try {
+            Endpoint endpoint = endpointRepository.getOrCreateDefaultEmailSubscription(action.getAccountId(), action.getOrgId());
+
+            // TODO Add recipients processing from policies-notifications processing (failed recipients)
+            //      by checking the NotificationHistory's details section (if missing payload - fix in WebhookTypeProcessor)
+
+            // TODO If the call fails - we should probably rollback Kafka topic (if BOP is down for example)
+            //      also add metrics for these failures
+
+            webhookSender.doHttpRequest(
+                event, endpoint,
+                bopRequest,
+                getPayload(user, action, subject, body), "POST", bopUrl, persistHistory);
```

Comment:
Can you please put all the arguments in new lines?

Reply:
Sure :)